### ISSUE TITLE
Integration doesnt show up on integration page

### ISSUE THREAD

**Issue by kobioneil** [fuatakgun/eufy_security#197]
Please check Services, Known Bug / Issues and Troubleshooting over here first: https://github.com/fuatakgun/eufy_security/blob/master/README.md

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Hardware Information about Home Assistant

**Hardware Information about Cameras
- Camera Model
- Live Streaming Protocol: RTSP or P2p

**Additional context**

**Comment by verwirrrung** [comment]
I have the same issue. Here is what I have...
- Successfully added Eufy Security Add-on and RTSP Simple Server Add-on
- I did not add the custom repo because HACS says that it is already there. 
![image](https://user-images.githubusercontent.com/71836292/148127383-e1812f59-f2db-418c-8684-56336f3d5300.png)

- So I added integration offered by HACS and did the HA restart. 
![image](https://user-images.githubusercontent.com/71836292/148128000-98f7ec69-7f9d-4de1-a021-0dac90c54ac6.png)

- I am now trying to add the integration by going to configuration --> Devices and Services --> Add integration button, but it is not listed there
![image](https://user-images.githubusercontent.com/71836292/148127865-6406dd96-7c0d-4840-836c-bc4f71182ec0.png)

Some things to note:
I amin the latest release of HA (December 2021) and my doorbell is in my IoT VLAN and my HA is in my Prod VLAN. Thoughts?
